### ISSUE TITLE
Unable to operate blinds since 2022-06-27 - Is there a max limit on the number of calls in a specific timeframe?

### ISSUE THREAD

**Issue by karanvivekbhargava** [mtgeekman/Home_Assistant_NeoSmartBlinds#27]
I had recently added blinds connected via this component (working perfectly well) to node red. And I might've been calling the blinds at about 2 Hz or in 30 sec increments to keep closing. Now the blinds aren't working with home assistant calls even though I've removed the continuous calls from nodered.

**Comment by karanvivekbhargava** [comment]
Solved: 
I restarted my neo smart blinds hub and then rebooted home assistant and was able to get them to talk again. Thank you for the component!

**Comment by gee-jay-bee** [comment]
Just to say that I have observed similar behaviour where the hub can become overloaded and unresponsive. More regularly, I see that successive calls can be ignored but the hub recovers after a little while.

I have a branch in this repo that imposes a minimum interval between calls which works ok. I set it to 500ms which is apparently the min period. It does help. Interestingly, a developer working on the hub (active on discord) couldn't understand why it helps as much as it does! He hasn't been able to investigate yet
